stories/panel.stories.js

```diff
@@ -88,7 +88,7 @@ storiesOf('Panel', module)
     withHits(({ search, container, instantsearch }) => {
       const brandList = instantsearch.widgets.panel({
         collapsed: options => {
-          return options && options.state && options.state.query.length === 0;
+          return !((options && options.state && options.state.query) || '');
```

Comment:
This is slightly more complicated. Would this make sense?

```suggestion
          return (options && options.state && (!options.state.query || options.state.query === ''));
```

Reply:
https://github.com/algolia/instantsearch.js/pull/3804/commits/80418777984a5366e30066b693d9098b39a715b6